plugins/defaultplugins/l3plugin/model/l3/l3.proto

```diff
@@ -3,17 +3,17 @@ syntax = "proto3";
 package l3;
 
 message StaticRoutes {
-    message Ip {
+    message Route {
         uint32 vrf_id = 1;                   /* VRF identifier */
```

Comment:
think about that something is in ETCD (misconfiguration) key contains VRF=0 but value contains VRF=1.

Since we would be incosistent with other impl. if we would remove the VRF field from the value we need at least overwrite it.